src/ThisAssembly.Resources/ThisAssembly.Resources.targets

```diff
@@ -0,0 +1,43 @@
+<Project>
+
+  <PropertyGroup>
+    <EmbeddedResourceStringExtensions>.txt;.cs;.sql;.json;.md;</EmbeddedResourceStringExtensions>
```

Comment:
In order to make this property fully customizable by users in their projects (including making it blank), it should go in a .props instead, I think.

Reply:
Makes sense - gonna be honest; didn't know there was a difference between the two until you commented and I did some research. :)